locale/en/blog/release/v6.0.0.md

```diff
@@ -0,0 +1,172 @@
+---
+date: 2016-04-26T20:35:53.135Z
+version: 6.0.0
+category: release
+title: Node v6.0.0 (Current)
+slug: node-v6-0-0
+layout: blog-post.hbs
+author: James M Snell
+---
+
+It's here! Just three days shy of exactly six months after we released Node v5.0.0, the Node.js developers are happy to announce that Node.js v6 is available and ready for use! 
+
+This new major version of Node.js delivers significant improvements in performance, reliability, usability and security. Module loading is nearly four times faster than in Node.js v4, tests and documentation have seen significant improvements, usability of the Buffer and File System APIs has been improved, and Node.js now supports 93% of JavaScript language features thanks to V8 version 5.0.
```

Comment:
@jasnell yeah, but i think it's still vague. and since es6 is the latest version, can we rephrase it into something like `93% of JavaScript/ES6 language features`?